src/inverseRobot.js

```diff
@@ -25,6 +25,24 @@
  */
 function inverseRobot(robot) {
   // write code here
+  let reversed = {};
+  const arrayOfProp = [];
+
+  for (const prop in robot) {
+    arrayOfProp.push(robot[prop]);
+
+    if (robot.hasOwnProperty(prop)) {
```

Comment:
prop that's a property which you just pull out of the robot, what exactly you want to get form this checking?

Reply:
Got it. I thought that I need to check if the object has the property which I'm going to change.  
But I loop through these properties before, so they are in the object. 
Then this condition is redundant.